### PR TITLE
feat(workflow): Amend GroupInbox model with organization and project

### PR DESCRIPTION
**Pull request by iProgramStuff**
The GroupInbox model needs some additional fields so that we can filter down to a smaller set of ids when performing searches.

This PR adds an `organization` and `project` field to the model.

### CODE REVIEW

**Comment by wedamija** [comment]
Since this table is reasonably large already, we'll probably want to create the indexes concurerntly. Example here: https://github.com/getsentry/sentry/blob/4f38b38d303d381453cb329cb04c730e6c2ff1b3/src/sentry/migrations/0115_add_checksum_to_debug_file.py#L18

File: src/sentry/models/groupinbox.py [hunk 1]
```diff
@@ -23,6 +23,9 @@ class GroupInbox(Model):
     __core__ = False
 
     group = FlexibleForeignKey("sentry.Group", unique=True, db_constraint=False)
+    project = FlexibleForeignKey("sentry.Project", null=True)
+    organization = FlexibleForeignKey("sentry.Organization", null=True)
```
Comment (wedamija) [inline src/sentry/models/groupinbox.py:27]: Let's use `db_constraint=False` here to be safe

File: src/sentry/migrations/0121_groupinboxprojectandorg.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-11-03 18:08
+from __future__ import unicode_literals
+
+from django.db import migrations
+import django.db.models.deletion
+import sentry.db.models.fields.foreignkey
+
+
+def obliterate_group_inbox(apps, schema_editor):
```
Comment (markstory) [inline src/sentry/migrations/0121_groupinboxprojectandorg.py:10]: Good function name.

File: src/sentry/migrations/0121_groupinboxprojectandorg.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = True
+
+
+    dependencies = [
+        ('sentry', '0120_commit_author_charfield'),
+    ]
+
+    operations = [
+        migrations.RunPython(obliterate_group_inbox, migrations.RunPython.noop),
```
Comment (markstory) [inline src/sentry/migrations/0121_groupinboxprojectandorg.py:39]: This table no longer has any write traffic so there won't be an contention on this table. 👍

**Review by markstory — APPROVED** [review]

**Pull request merged**